docs/sphinx/developer/docker.rst

```diff
@@ -152,7 +154,9 @@ Test & Develop
 
 The Dockerfile is defined to start all Phoniebox related services.
 
-Open `http://localhost:3001 <http://localhost:3001>`_ in your browser to see the web application.
+Windows: Open `host.docker.internal:3000 <http://host.docker.internal:3000>`_ in your browser to see the web application.
+Mac: Open `host.docker.internal:3000 <http://host.docker.internal:3000>`_ in your browser to see the web application.
```

Comment:
At least in Mac, it only works with `localhost:3000`

Reply:
OK good to know, then I'll change it back.
Do you know if that host entry host.docker.internal exists on Mac?

Reply:
See above